tests/wicked/before_test.pm

```diff
@@ -133,16 +133,16 @@ sub run {
             zypper_ar($repo_url . generate_version('_') . '/', name => 'wicked_maintainers', no_gpg_check => 1, priority => 60);
             $package_list .= ' ndisc6';
         }
-        if (check_var('WICKED', 'wlan')) {
-            wicked::wlan::before_test();
-        }
+        wicked::wlan::before_test() if (check_var('WICKED', 'wlan'));
```

Comment:
```suggestion
        wicked::wlan::prepare_wlan_packages() if (check_var('WICKED', 'wlan'));
```

Reply:
with deletion of `before_test` function you don't need it anymore

Reply:
ok